corehq/apps/locations/dbaccessors.py

```diff
@@ -92,3 +92,23 @@ def user_ids_at_accessible_locations(domain_name, user):
     from corehq.apps.locations.models import SQLLocation
     accessible_location_ids = SQLLocation.active_objects.accessible_location_ids(domain_name, user)
     return user_ids_at_locations(accessible_location_ids)
+
+
+def assigned_location_by_user_id(domain, location_ids):
```

Comment:
not sure i understand why this is necessary - this looks like it's getting all the locations that the users are assigned to, by searching on primary location id? i think it's safe to only use a user's primary location for these indicators